recipes/linalg-complex/24.11/gh200/environments.yaml

```diff
@@ -10,6 +10,7 @@ gcc-env:
   - arpack-ng
   - boost +atomic +chrono +container +context +coroutine +date_time +filesystem +iostreams +json +log +mpi +multithreaded +program_options +python +random +regex +serialization +shared +system +test +thread +timer +url
   - cmake
+  - dla-future +miniapps +mpi_gpu_aware +mpi_gpu_force_contiguous +scalapack
```

Comment:
The spec effectively matches what is set in https://github.com/eth-cscs/alps-uenv/pull/187 (`+scalapack` is implied by cp2k, `+mpi_gpu_force_contiguous` is the default, but I've preferred the explicit version here).

Reply:
Because of `views:link:roots` you will need to add several root spec in order to be able to use pika and DLAF. This is what I added to the CP2K UENV:
```
  # pika
  - pika@0.30.1 malloc=mimalloc
  - mimalloc
  - boost
  - fmt
  - spdlog
  - hwloc
  - whip
  # Umpire
  - umpire
  - blt
  - camp
  # DLA-Future
  - dla-future@0.7.3 +mpi_gpu_aware +mpi_gpu_force_contiguous +scalapack +miniapps
  - blaspp
  - lapackpp
  - dla-future-fortran
```
I can't guarantee that this is the minimal set of root specs needed, because I lost patience when adding them one-by-one. For pika and DLAF they correspond to CMake's `find_dependencies`. For `umpire` I'm not sure `blt` and `camp` are actually needed, but I added them just in case.